nixui/options/option_tree.py

```diff
@@ -256,6 +272,13 @@ def children(self, attribute, mode="direct"):
             if '"<name>"' not in node.tag
         }
 
+    @functools.lru_cache(100000)  # this breaks when nixos has 100,000 attributes
```

Comment:
> this breaks when nixos has 100,000 attributes

so its not a "cache" but a store?

Reply:
Yes

Reply:
I've merged a few PRs recently, if you'd like to review them you're more than welcome to so I can make any necessary changes. They're all relatively low risk though.